src/njoy21/input/LEAPR/Card17/Dka.hpp

```diff
@@ -0,0 +1,19 @@
+struct Dka {
+  using Value_t = decltype( pow( 1.0 * angstrom, Ratio<-1> ) );
```

Comment:
I'm not sure what this does. I think it gets the type from a value of 1Å^(-1). I suppose that would be inverse angstroms.